gluon/custom_import.py

```diff
@@ -86,7 +86,7 @@ def custom_importer(name, globals=None, locals=None, fromlist=None, level=-1):
                         try:
                             result = result or sys.modules[modules_prefix+'.'+itemname]
                         except KeyError as e:
-                            raise ImportError, 'Cannot import module %s' % str(e)
+                            raise ImportError('Cannot import module %s' % str(e))
```

Comment:
Did we address byte/string/unicode of py2-py3? If so, is str(e) here will cause issue?